meta-facebook/yv35-cl/src/platform/plat_isr.c

```diff
@@ -118,6 +125,34 @@ void ISR_PWRGD_CPU()
 	send_gpio_interrupt(PWRGD_CPU_LVC3);
 }
 
+static void CatErr_handler(void *arug0, void *arug1, void *arug2)
+{
+	if ((gpio_get(RST_PLTRST_BUF_N) == 1) || (gpio_get(PWRGD_SYS_PWROK) == 1)) {
+		addsel_msg_t sel_msg;
+		bool ret = 0;
```

Comment:
bool should be set to true/false, not zero.

Reply:
We have fixed it and done the forced update.
Please review it. Thanks.